src/main/kotlin/com/ing/dlt/zkkrypto/ecc/schnorr/SchnorrSignature.kt

```diff
@@ -0,0 +1,162 @@
+package com.ing.dlt.zkkrypto.ecc.schnorr
+
+import com.ing.dlt.zkkrypto.ecc.EllipticCurve
+import com.ing.dlt.zkkrypto.ecc.EllipticCurvePoint
+import com.ing.dlt.zkkrypto.ecc.curves.AltBabyJubjub
+import com.ing.dlt.zkkrypto.ecc.schnorr.FixedGenerators.altBabyJubjubFixedGenerators
+import com.ing.dlt.zkkrypto.util.asUnsigned
+import org.bouncycastle.crypto.digests.Blake2bDigest
+import org.bouncycastle.crypto.digests.Blake2sDigest
+import java.math.BigInteger
+import java.security.SecureRandom
+import kotlin.random.Random.Default.nextBytes
+
+enum class Hash{
```

Comment:
is this not already defined somewhere in the lib? because Alexei did one of this hashes.

Reply:
It's not, afaik. But I removed this outside of this class and defined as an enum with other supported hash functions. Because, in the future, Mimc or Poseidon can be used here.